packages/flutter/test/material/about_test.dart

```diff
@@ -593,6 +594,138 @@ void main() {
     expect(nestedObserver.dialogCount, 1);
   });
 
+  group('showAboutDialog avoids overlapping display features', () {
+    testWidgets('default positioning', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+        builder: (BuildContext context, Widget? child) {
+          return MediaQuery(
+            // Display has a vertical hinge down the middle
+            data: const MediaQueryData(
+              size: Size(800, 600),
+              displayFeatures: <DisplayFeature>[
+                DisplayFeature(
+                  bounds: Rect.fromLTRB(390, 0, 410, 600),
+                  type: DisplayFeatureType.hinge,
+                  state: DisplayFeatureState.unknown,
+                ),
+              ],
+            ),
+            child: child!,
+          );
+        },
+        home: Builder(
+            builder: (BuildContext context) => ElevatedButton(
+              onPressed: () {
+                showAboutDialog(
+                  context: context,
+                  useRootNavigator: false,
+                  applicationName: 'A',
+                );
+              },
+              child: const Text('Show About Dialog'),
+            )
+        ),
+      ),
+      );
+
+      // Open the dialog.
+      await tester.tap(find.byType(ElevatedButton));
+      await tester.pumpAndSettle();
+
+      // By default it should place the dialog on the left screen
+      expect(tester.getTopLeft(find.byType(AboutDialog)), Offset.zero);
+      expect(tester.getBottomRight(find.byType(AboutDialog)), const Offset(390.0, 600.0));
+    });
+
+    testWidgets('positioning using anchorPoint', (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(
+        builder: (BuildContext context, Widget? child) {
+          return MediaQuery(
+            // Display has a vertical hinge down the middle
+            data: const MediaQueryData(
+              size: Size(800, 600),
+              displayFeatures: <DisplayFeature>[
+                DisplayFeature(
+                  bounds: Rect.fromLTRB(390, 0, 410, 600),
+                  type: DisplayFeatureType.hinge,
+                  state: DisplayFeatureState.unknown,
+                ),
+              ],
+            ),
+            child: child!,
+          );
+        },
+        home: Builder(
+            builder: (BuildContext context) => ElevatedButton(
+              onPressed: () {
+                showAboutDialog(
+                  context: context,
+                  useRootNavigator: false,
+                  applicationName: 'A',
+                  anchorPoint: const Offset(1000, 0),
+                );
+              },
+              child: const Text('Show About Dialog'),
+            )
```

Comment:
nit: the indentation appears off here. Add a trailing comma here and unindent the previous lines by 2.